src/feature/hs/hs_client.c

```diff
@@ -1681,62 +1892,39 @@ hs_config_client_authorization(const or_options_t *options,
   return ret;
 }
 
-/** This is called when a descriptor has arrived following a fetch request and
- * has been stored in the client cache. Every entry connection that matches
- * the service identity key in the ident will get attached to the hidden
- * service circuit. */
+/** Called when a descriptor directory fetch is done. The ident is the
```

Comment:
Let's improve  func doc here by saying what this function does and not only when it's called (or what it's arguments are).

For example, its calls are tasked with setting the SOCKS extended error code but this is not documented neither here or on its calls function documentation.

Reply:
Fixup commit: 3aae107e8e